src/workerd/api/node/tests/http-server-nodejs-test.js

```diff
@@ -1132,6 +1132,28 @@ export const testConfigurableHighWaterMark = {
   },
 };
 
+export const testIncomingMessageSocket = {
+  async test(_ctrl, env) {
+    await using server = http.createServer((req, res) => {
+      strictEqual(req.socket.encrypted, false);
+      strictEqual(req.socket.localPort, 8080);
+      strictEqual(req.socket.localAddress, '127.0.0.1');
+      strictEqual(req.socket.remoteAddress, '127.0.0.1');
+      strictEqual(typeof req.socket.remotePort, 'number');
```

Comment:
Per the other comments, if `remotePort` is expected to be within a specific range we should likely test for that.